Yosemite/Yosemite/Model/Extensions/User+Roles.swift

```diff
@@ -1,7 +1,7 @@
 import Foundation
 
 /// Convenient extension methods for User model to support role eligibility feature.
-extension User {
+extension User {//
```

Comment:
```suggestion
extension User {
```

Reply:
Must have commited by accident.. I sometimes put // in files while I debug so I can easily just look at "changed" files and quickly switch between.. but I must have forgot to discard this in commit.. tnx!

Reply:
> I sometimes put // in files while I debug so I can easily just look at "changed"

I do the same 😁